server/src/main/java/org/moparforia/server/db/Database.java

```diff
@@ -11,7 +11,7 @@
 // * Date: 2013-08-05
 // * Time: 12:13
 // */
-//public class Database {
+public class Database {
```

Comment:
Not sure why this is changed? The class seems to be not doing anything

Reply:
When this was fully commented, Maven was recompiling the entire source
every time it was run because the file was invalid.
See https://stackoverflow.com/questions/17944108/maven-compiler-plugin-always-detecting-a-set-of-sources-as-stale for more information.

Adding this class back speeds up the iteration cycle of server development by a few seconds.

Reply:
Interesting, how about removing it instead? I do not see any value in that class